maestrowf/abstracts/interfaces/schedulerscriptadapter.py

```diff
@@ -95,7 +94,7 @@ def get_header(self, step):
         pass
 
     @abstractmethod
-    def get_parallelize_command(self, procs, nodes=None):
+    def get_parallelize_command(self, nodes, proc, **kwargs):
```

Comment:
You swapped the order of the arguments `nodes` and `proc`, but you use them in the original order below.

Reply:
That was unintentional. Fixed.